styleguide/components/_preview-description-list.twig

```diff
@@ -1,51 +1,13 @@
-<!DOCTYPE html>
```

Comment:
I like this, but maybe we should add a comment section to the different preview templates to make clear when to use them for other or new developers.

Reply:
Yeah, not a bad idea!

Reply:
👍